readme.md

```diff
@@ -1,44 +1,95 @@
 ## Repository Purpose:
-This repository contains code used to generate synthetic LAR files. These files will be used to support the development of the 2018 HMDA data submission platform. The success of the HMDA program requires that the data submitted meet the technical and business requirements of the HMDA Platform. The code and test files in this repository will provide a standardized means of checking the implementation of the business rules for data submission in the HMDA Platform.
+This repository contains code used to generate synthetic LAR files. These files will be used to support the development of the 2018 HMDA data submission platform and platforms in the future. The success of the HMDA program requires that the data submitted meet the technical and business requirements of the HMDA Platform. The code and test files in this repository will provide a standardized means of checking the implementation of the business rules for data submission in the HMDA Platform.
 
 ## Repository Use Cases:
 - Enable business analysts to test the 2018 HMDA Platform
 - Provide files to check API returns
 - Provide a resource for developers to use while implementing business rules in code
 - Provide synthetic data files meeting the 2018 HMDA schema to be used in training
 
+## Repository Structure
+- [Python](https://github.com/cfpb/hmda-test-files/tree/master/python):
+    - Contains the notebooks and python scripts used to generate synthetic LAR data, perturb the data so that it fails specific business rules, and code to check that the produced files fail as expected.
+
+- [Schemas](https://github.com/cfpb/hmda-test-files/tree/master/schemas)
+    - Contains JSON objects that represent the data structures for LAR and Transmittal Sheet as defined by the [2018 HMDA FIG](https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2018-hmda-fig-2018-hmda-rule.pdf)
+
+- [Dependencies](https://github.com/cfpb/hmda-test-files/tree/master/dependencies)
+    - Contains files used in the generation of synthetic LAR data.
+        - Tract to CBSA data for 2015 (most current year)
```

Comment:
This data was updated a few times, ours is old and needs an update here.
Can you specify what is currently being used and link to the file?

Reply:
Sure! I can take out the parenthetical statement. If it is okay, we may want to add the new files and then update this documentation in another issue.